docs/faq.md

```diff
@@ -0,0 +1,39 @@
+---
+hide:
+  - navigation
+---
+
+# Frequently Asked Questions
+
+## General
+
+### What kinds of apps is Mesop suited for?
+
+Mesop is particularly well-suited for ML/AI demos and internal tools because it enables developers without frontend experience to quickly build web apps. For use cases that prioritize developer experience and velocity, Mesop can be a good choice.
+
+For apps with stringent demands, either in terms of performance or UI customization, then other UI frameworks may be more suitable.
```

Comment:
I feel like "UI customization" could be misconstrued here. I feel like you can heavily customize the UI using Styles. There are certainly limitations in the styling and also we don't have all the Angular components and we don't have a way for users to create custom components yet. So maybe need to be more clear.

I think the issue could be the use of "stringent". People may take a stringent in different ways. I feel like the better word may be something like "production". At the moment, Mesop is definitely not something people would want to run in a serious production application yet.

Reply:
OK, I reworded "ui customization". I don't know if "production" is better because we are running things like Demo Gallery, which I would argue is a Mesop app in production (e.g. it has external traffic w/ good availability, etc.).

In my mind, it's basically - don't try to run a consumer-grade app e.g. with millions of users which requires a11y, i18n, SEO-optimization, etc. I've rephrased this below